packages/jaspr_lints/lib/src/all_html_tags.dart

```diff
@@ -0,0 +1,70 @@
+const allHtmlTags = {
+  'article',
+  'aside',
+  'body',
+  'footer',
+  'head',
+  'header',
+  'html',
+  'h1',
+  'h2',
+  'h3',
+  'h4',
+  'h5',
+  'h6',
+  'main_',
```

Comment:
Is this bug? I don't know if should be 'main' or 'main_' here..

Reply:
No, thats the method name to not conflict with `main()`.
This file is auto generated btw.